settings.py

```diff
@@ -258,8 +258,8 @@
         'http://localhost:3000',
         'http://localhost:8000',
     )
-    # STATICFILES_DIRS.append(STATIC_ROOT)
-    # STATIC_ROOT = None
+    STATICFILES_DIRS.append(STATIC_ROOT)
```

Comment:
my static assets didn't load with this commented out... was there maybe a reason?

Reply:
were you running localhost:8000 or localhost:3000 ?

Reply:
I think the reason we left these commented out is if you want to use the built assets and localhost:8000 you have to uncomment them. But if you want to be testing using the live javascript assets on localhost:3000, which is how we basically always want to do locally since the built assets in the seqr repo are not necessarily current, then they need to be commented out

Reply:
I'm running the js on `localhost:3000`, which is proxying to the Django server on 8000.  The dev server live updating seems to be working.  

The issue is isolated to the icons in `seqr/static` itself.   I don't have any actually built javascript in `ui/dist`.

Reply:
For future reference, this breaks prod deploys which is why we've always kept it commented out 🤦 
https://github.com/broadinstitute/seqr/actions/runs/6435806250/job/17477833506